templates/tfengine/components/app/devops/main.tf

```diff
@@ -45,14 +45,13 @@ locals {
 # Create the project, enable APIs, and create the deletion lien, if specified.
 module "project" {
   source  = "terraform-google-modules/project-factory/google"
-  version = "~> 9.2.0"
+  version = "~> 10.0.1"
   name                    = "${local.constants.project_prefix}-${local.constants.env_code}-devops"
   org_id                  = ""
   folder_id               = local.constants.folder_id
   billing_account         = local.constants.billing_account
   lien                    = true
   default_service_account = "keep"
-  skip_gcloud_download    = true
```

Comment:
How come this is being removed?

Reply:
They now use a gcloud module to achieve what was done by `gcloud` commands. So this attribute does not apply anymore and is removed. Also see https://github.com/terraform-google-modules/terraform-google-project-factory/blob/master/docs/upgrading_to_project_factory_v10.0.md#migration-instructions

Reply:
Cool, that's good to know!